storage/replica_command.go

```diff
@@ -2336,6 +2336,9 @@ func (r *Replica) AdminSplit(
 			return err
 		}
 		if err := txn.Run(b); err != nil {
+			if _, ok := err.(*roachpb.ConditionFailedError); ok {
+				return errors.Errorf("conflict updating range descriptors")
```

Comment:
Yeah, but do we need to include the original error. Using `errors.Wrapf` the error prints as:

```
expected error: storage/replica_command.go:2368: split at key "a" failed: conflict updating range descriptors: unexpected value: raw_bytes:"I\205B\247\003\010\001\022\000\032\001a\"\006\010\001\020\001\030\001(\002" timestamp:<wall_time:123 logical:70 >
```

vs `errors.Errorf`:

```
expected error: storage/replica_command.go:2368: split at key "a" failed: conflict updating range descriptors
```